.github/workflows/gui-changed-files.yml

```diff
@@ -37,7 +37,6 @@ jobs:
             .prettierignore
             vitest.workspace.ts
             .github/workflows/gui*
-            .github/workflows/ide*
             .github/workflows/storybook.yml
           files_ignore: |
             app/ide-desktop/**
```

Comment:
Because `gui-changed-files.yml` ignores the IDE directory, this PR introduces a separate `ide-changed-files.yml` that watches Electron-related files and triggers the _IDE Packaging_ workflow.